02-about.Rmd

```diff
@@ -0,0 +1,80 @@
+
+### MI (Management Information) systems
+
+MI is data collected and used by a service team to track and manage their service day-to-day. In contrast AP has been designed for analysts who produce official statistics and generally work on longer term decision-making with policy teams. There is a view that the user needs differ between these two groups - a service team's business/performance analyst might look at standard performance metrics, suited to standard business spreadsheet or BI software. In contrast, an analyst supporting policy decisions tends to use complex methods that are more suited to being expressed in code, and require more formal methods to ensure quality, such as reproducibility.
+
+In addition, MI is something that nearly every team in the department needs to do, is likely to be closely coupled with the service's systems and business needs. Understanding this environment and user needs across the department is not something that has been the focus of AP. So whilst there is considerable overlap - data storage, processing and dashboards - AP has not been designed for this environment. Indeed there is resistance to adding point-and-click BI software to AP, since the analysts are being encouraged to work in a reproducible way.
+
+### Real time data
+
+AP has Airflow, which can schedule data flows and processing as frequently as every few minutes. However as you move towards on-going data processing like this you start to benefit from data streaming technology, to provide real-time results, cope with disturbances in the flow rate, manage processing in small batches etc. 
+
+Technically there is a clear route to enabling data streaming on AP, such as with Apache Kafka or managed cloud services. However it has not yet been explored yet.
+
+### Data archival
+
+Data often needs to be retained safely for long periods, for possible use in the future. On one hand, AP offers an assured place for storing data, with durability, backups, security and access controls. However brief analysis shows some archival needs not fully met:
+
+* Archival tends to require an index and search facility. S3 doesn't offer these, so a way to extract text from files and provide an index & search interface would need to be found.
+* S3 is not the cheapest storage, although a custom bucket could be set-up with a policy to archive it to S3-IA or Glacier.
+
+SaaS alternatives, typified by Sharepoint or Google Drive, are often more suited to archival.
+
+## Who it is for
+
+AP is primarily for analysts and data scientists within the Ministry of Justice, including CICA, HMCTS, HMPPS, LAA and OPG. Access for other justice organizations is considered - [contact us](mailto:analytical_platform@digital.justice.gov.uk) to discuss.
```

Comment:
It is primarily for analysts in the Data and Analytical Services Directorate in the Ministry of Justice. We also have users from CICA, HMCTS, HMPPS, LAA, and OPG, but they are not core users (the external users policy hopefully clarifies this - the distinction is that we give less weight to their feature requests, but that probably needn't be stated here).